dagql/idtui/frontend_pretty.go

```diff
@@ -446,16 +626,58 @@ func (fe *frontendPretty) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			fe.restore()
 			sigquit()
 			return fe, nil
+		case "down", "j":
+			fe.goDown()
+			return fe, nil
+		case "up", "k":
+			fe.goUp()
+			return fe, nil
+		case "left", "h":
+			fe.goOut()
+			return fe, nil
+		case "right", "l":
+			fe.goIn()
+			return fe, nil
+		case "home":
+			fe.goStart()
+			return fe, nil
+		case "end", " ":
+			fe.goEnd()
+			return fe, nil
+		case "esc":
+			fe.zoomed = fe.db.PrimarySpan
+			fe.recalculateViewLocked()
+			return fe, nil
+		case "+":
+			fe.FrontendOpts.Verbosity++
+			fe.recalculateViewLocked()
+			return fe, nil
+		case "-":
+			fe.FrontendOpts.Verbosity--
+			if fe.FrontendOpts.Verbosity < 0 {
+				fe.FrontendOpts.Verbosity = 0
+			}
+			fe.recalculateViewLocked()
+			return fe, nil
+		case "c":
+			if fe.cloudURL != "" {
+				browser.OpenURL(fe.cloudURL)
+			}
+			return fe, nil
+		case "enter":
+			fe.zoomed = fe.focused
```

Comment:
Once we're zoomed, can we give some indication of the "context" we're in? Maybe like a breadcrumbs trail?

Reply:
Done! Now it'll pin your zoomed span to the top and nest the children, feels a lot more intuitive and avoids the feeling of having another UI element:

![image](https://github.com/dagger/dagger/assets/1880/1709ab07-1a41-4c0b-a520-010dc8396321)

Reply:
Yay :tada: feels like cloud now :tada: